pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -600,16 +602,19 @@ def __log_evaluation_epoch_metrics(self, eval_results, test_mode):
     def evaluation_forward(self, model, batch, batch_idx, dataloader_idx, test_mode: bool = False):
         # make dataloader_idx arg in validation_step optional
         args = [batch, batch_idx]
-
+        print('here 3')
         if (test_mode and len(self.test_dataloaders) > 1) \
                 or (not test_mode and len(self.val_dataloaders) > 1):
             args.append(dataloader_idx)
 
         # handle DP, DDP forward
         if self.use_ddp or self.use_dp or self.use_ddp2:
+            # SOMETHING GOES WRONG HERE, test loop is stuck
             output = model(*args)
             return output
```

Comment:
In the test loop, at batch 9, it get's stuck here. The cpu and gpu usage stays at 100% as if it is busy but it is just hanging here at the same batch. Any ideas? 
I also checked master, same problem there, so not related to my changes in this PR here.

Reply:
you get any errors messages that you can copy paste here?

Reply:
nope, it just hangs. this happens also on master. Here I just used stupid print statements to figure out where it occurs xD